.pre-commit-hooks.yaml

```diff
@@ -20,3 +20,9 @@
   entry: (has_perm|@permission_required)\(["'][^\."']+["']\)
   language: pygrep
   types: [python]
+#  when calling this hook in a repo you must add args: [<path to local settings e.g. qrhub.settings.local>] in the repo .pre-commit-config.yaml file
```

Comment:
It would be better if we could but I couldn't find a way to get it into the yaml file

Reply:
This needs to use the env var, otherwise it won't be able to use the correct settings in both local and CI contexts. Not sure what you mean by getting it into the YAML file, but first thing to try would be `echo $DJANGO_SETTINGS_MODULE` in the bash script to see if it's set.

Reply:
I tried that but will just double check